app/views/tasks/modify.html.erb

```diff
@@ -10,10 +10,11 @@
   <%= f.text_field :description %>
 
   <%= f.label :completed %>
-  <%= f.text_field :completed %>
+  <%= f.select :completed, options_for_select([['True', "t"], ['False', "f"]]) %>
+
+  <%= f.label :person_id %>
+  <%= f.select :person_id, options_for_select([['Michelle', 1], ['Lindsey', 2], ['Kari',3]]) %>
```

Comment:
Take a look at the `collection_select` which would allow you to avoid hard-coding the people into this drop-down: http://api.rubyonrails.org/classes/ActionView/Helpers/FormOptionsHelper.html